gradle.properties

```diff
@@ -18,5 +18,5 @@ ideaEdition = IC
 ideaVersion = 2017.1
 javaVersion = 1.8
 ijPluginRepoChannel = alpha
-version = 17.2.2_2017-SNAPSHOT
+version = 17.2.3
```

Comment:
Didn't we decide that for the next version, master would start targeting 2017?

If so, 17.2.3 (even 17.3) should be the next version.